src/styles/dialog.scss

```diff
@@ -175,4 +175,12 @@ $button: #{$fd-namespace}-button;
       color: $fd-dialog-resize-handle-color;
     }
   }
+
+  &__no-backdrop:not(.#{$block}__targeted) {
```

Comment:
Is it a modifier class or an element? Because from the naming it looks like you are introducing a new html element with the class `.fd-dialog__no-backdrop`. If it's a modifier class, added on the same level as `.fd-dialog`, then you should have a different naming, like `.fd-dialog--no-backdrop`.

Reply:
done